modules/google_sql_database_instance/main.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 4.0"
+      version = "~> 4.48.0"
```

Comment:
Why is this changing to now pin the minor version?  This will block non breaking provider updates in consuming modules.